src/styles/index.css

```diff
@@ -1,13 +1,20 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+@import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+#main-container {
```

Comment:
Do you want this code here? It might be a better idea to refactor this.

Reply:
I'm going to leave it for now, once I figure the CSS with Anvinh I'll go back to this.